modules/decline/src/core/commons.scala

```diff
@@ -52,9 +52,8 @@ object commons {
 
     Argument.from("json")(parse(_).toValidatedNel)
   }
-  val byteArrayArgument: Argument[ByteArray] = Argument.from("base64")(
-    s =>
-     Valid(ByteArray(s.getBytes("UTF-8"))))
+  val byteArrayArgument: Argument[ByteArray] =
+    Argument.from("base64")(s => Valid(ByteArray(s.getBytes("UTF-8"))))
```

Comment:
**thought:** should we decode the base64 to a plain array of bytes before passing it to the ByteArray?

Reply:
yes it should